engine/src/flutter/shell/platform/fuchsia/dart/BUILD.gn

```diff
@@ -106,7 +106,7 @@ dart_library("args") {
   package_root = "$dart_src/third_party/pkg/core/pkgs/args"
   package_name = "args"
 
-  language_version = "2.17"
+  language_version = "3.7"
```

Comment:
It's really unfortunate that we have hardcoded language_version here since has been consistent source of issues in the past. I wonder if there is better solution. I know we used `pubspec = "$package_root/pubspec.yaml"` in the past.